bin/ember

```diff
@@ -7,6 +7,11 @@ process.title = 'ember';
 var resolve = require('resolve');
 var exit = require('exit');
 
+var updateNotifier = require('update-notifier');
```

Comment:
Yes, that makes sense. I think this is the correct direction now, we just need to make sure the UX around this isn't confusing.